### PR TITLE
User should be able to view Steps of a Chapter on Chapter view page #174

### PR DESCRIPTION
**Pull request by bbiehl**

### CODE REVIEW

File: app/controllers/chapters_controller.rb [hunk 1]
```diff
@@ -30,6 +30,8 @@ def update_confirmation
   end
 
   def view
+    @tutorial =Tutorial.find_by_id(params[:tutorial_id])
```
Comment (sumeetjain) [inline app/controllers/chapters_controller.rb]: Surrounding space missing for operator `=`.

**Comment by bbiehl** [comment]
duh, this is because the variable for @chapter was already defined in the class.

File: app/controllers/steps_controller.rb [hunk 2]
```diff
@@ -15,22 +15,14 @@ def create_confirmation
...
   end
-
-
-
-
-
-
-
-
+
```
Comment (sumeetjain) [inline app/controllers/steps_controller.rb]: Remove trailing whitespace

Comment (sumeetjain) [inline app/controllers/steps_controller.rb]: Trailing whitespace detected.

File: config/routes.rb [hunk 3]
```diff
@@ -32,6 +32,8 @@
   get 'tutorials/:tutorial_id/chapters/create' => "chapters#create"
   post 'tutorials/:tutorial_id/chapters/create/confirmation'=> "chapters#create_confirmation"
 
+  get 'tutorials/:tutorial_id/chapters/:chapter_id/steps/:step_id' => "steps#view"
```
Comment (sumeetjain) [inline config/routes.rb:35]: Line is too long. [82/80]

File: config/routes.rb [hunk 4]
```diff
@@ -41,16 +43,13 @@
 
   get 'tutorials/:tutorial_id/chapters/:chapter_id' => "chapters#view"
 
-  get 'tutorials/:tutorial_id/chapters/:chapter_id' => "chapters#delete"
-  delete 'tutorials/:tutorial_id/chapters/:chapter_id' => "chapters#delete"
+  delete 'tutorials/:tutorial_id/chapters/:chapter_id/delete' => "chapters#delete", as: :delete_chapter
```
Comment (sumeetjain) [inline config/routes.rb:46]: Line is too long. [103/80]

File: config/routes.rb [hunk 5]
```diff
@@ -41,16 +43,13 @@
...
 
   get 'tutorials/:tutorial_id/chapters/:chapter_id/steps/:step_id' => "steps#view"
 
-  get 'tutorials/:tutorial_id/chapters/:chapter_id/steps/:step_id' => "steps#delete"
-  delete 'tutorials/:tutorial_id/chapters/:chapter_id/steps/:step_id' => "steps#delete"
+  delete 'tutorials/:tutorial_id/chapters/:chapter_id/steps/:step_id' => "steps#delete", as: :delete_step
```
Comment (sumeetjain) [inline config/routes.rb:50]: Line is too long. [105/80]

File: config/routes.rb [hunk 6]
```diff
@@ -41,16 +43,13 @@
...
 
-  get 'tutorials/:tutorial_id' => "tutorials#delete"
-  delete 'tutorials/:tutorial_id' => "tutorials#delete"
+  delete 'tutorials/:tutorial_id/delete' => "tutorials#delete", as: :delete_tutorial
```
Comment (sumeetjain) [inline config/routes.rb:58]: Line is too long. [84/80]

**Pull request merged**